README.md

```diff
@@ -63,6 +63,13 @@ Initially puppet deploys all configuration to
 If and only if successful the configuration will be copied to
 the real locations before the service is reloaded.
 
+## Un-managed rules
+
+Rules added manually by the administrator to the in-memory ruleset
+will be left untouched. However, `nftables::allow_unmanaged_rules` can
+be set to `false` to revert this behaviour and force a reload of the
+ruleset during the Puppet run if non-managed changes are detected.
```

Comment:
Should we add that this doesn't work for rulesets with counters (or does it?) and likely also not for dynamic rulesets via tools like fail2ban.

Reply:
Yup, it does work with counters as the hash is generated from the "stateless" representation of the rules. However we indeed have to decide what to do if `noflush_tables` is set.

Reply:
Let's continue in https://github.com/voxpupuli/puppet-nftables/pull/115#discussion_r764968307, it's the same conversation.